docs/operating/reverse-proxy.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Running Thanos behind a reverse proxy
+type: docs
+menu: operating
+slug: /reverse-proxy.md
+---
+
+# Running Thanos behind a reverse proxy
+
+There are many reasons to use a [reverse proxy](https://www.nginx.com/resources/glossary/reverse-proxy-server/) in front of Thanos. For example, SSL termination (serve Thanos over HTTPS) and basic authentication. This small guide will tell you how to do that.
```

Comment:
Nit, the sentence starting with "For example" has no independent clause, ie it's not a full sentence. Let's join it with the previous sentence

Reply:
Nit s/SSL/TLS/

Reply:
Sure :+1: .